lib/src/mui_suggestors/mui_importer.dart

```diff
@@ -16,11 +16,9 @@ import 'package:analyzer/dart/ast/ast.dart';
 import 'package:analyzer/source/line_info.dart';
 import 'package:codemod/codemod.dart';
 import 'package:collection/collection.dart';
-import 'package:logging/logging.dart';
```

Comment:
This is now deleting some unused loggers. I think we should not do that for things outside of the intl_suggestors.

Reply:
Reverted code added `import 'package:logging/logging.dart';` here